### ISSUE TITLE
Documentation to update versions

### ISSUE THREAD

**Issue by traseable-solutions** [hzi-braunschweig/SORMAS-Docker#340]
Hi there,

Is there any documentation on how to upgrade to a newer version of SORMAS-Docker?

We're trying to upgrade a production version which is quite behind but don't exactly know how to do this.

I see that the main SORMAS-Project has good documentation on doing upgrades.

**Comment by Candice-Louw** [comment]
Initial feedback from @JonasCir -
From what I expect it would be something like pulling the new git tag, creating a backup from the DBs you care about, and run docker(-)compose down and then docker compose up -d --force-recreate.

**Comment by Candice-Louw** [comment]
@traseable-solutions - could you please share your feedback on whether the mentioned directions worked for you or if you did something else to handle the problem?  If you post the feedback here, it would assist the wider community in knowing what to do in future.

**Comment by JonasCir** [comment]
I'm going to leave this open as I couldn't find any documentation about this in the docs of this repo and it is one of the basic maintenance workflows. Docs are really necessary for this :monocle_face:

**Comment by traseable-solutions** [comment]
@Candice-Louw yes, I believe those are the correct steps you mentioned but I'm encountering errors when I do `docker-compose up` - the **apache2** container always fails.

Currently working through that issue and will share my results here once I get it working.
